pkgs/tools/misc/tldr/default.nix

```diff
@@ -11,9 +11,14 @@ stdenv.mkDerivation rec {
     sha256 = "10ylpiqc06p0qpma72vwksd7hd107s0vlx9c6s9rz4vc3i274lb6";
   };
 
-  buildInputs = [ curl clang libzip ];
+  buildInputs = [ git curl clang libzip ];
   nativeBuildInputs = [ pkgconfig ];
 
+  preConfigure = stdenv.lib.optionalString stdenv.isDarwin ''
+    substituteInPlace Makefile --replace "gcc" "$CC"
```

Comment:
I think you can also do ```makeFlags = [ "CC=$CC" ];```

Reply:
I tried and can't get it work using `makeFlags` :/ Thanks!